ambari/src/main/java/io/brooklyn/ambari/AmbariClusterImpl.java

```diff
@@ -46,28 +38,19 @@
 import brooklyn.entity.basic.SoftwareProcess;
 import brooklyn.entity.group.DynamicCluster;
 import brooklyn.entity.proxying.EntitySpec;
-import brooklyn.entity.software.ssh.SshCommandSensor;
 import brooklyn.event.SensorEvent;
 import brooklyn.event.SensorEventListener;
 import brooklyn.location.Location;
-import brooklyn.location.jclouds.BasicJcloudsLocationCustomizer;
-import brooklyn.location.jclouds.JcloudsLocation;
-import brooklyn.location.jclouds.JcloudsLocationConfig;
-import brooklyn.location.jclouds.JcloudsLocationCustomizer;
-import brooklyn.location.jclouds.JcloudsSshMachineLocation;
-import brooklyn.location.jclouds.networking.JcloudsLocationSecurityGroupCustomizer;
-import brooklyn.util.config.ConfigBag;
-
-import com.google.common.base.Joiner;
-import com.google.common.base.Optional;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.primitives.Ints;
+import io.brooklyn.ambari.agent.AmbariAgent;
+import io.brooklyn.ambari.agent.AmbariAgentImpl;
+import io.brooklyn.ambari.hostgroup.AmbariHostGroup;
+import io.brooklyn.ambari.rest.AmbariConfig;
+import io.brooklyn.ambari.server.AmbariServer;
 
 /**
-                "minRam", 8192,
-                "osFamily", "ubuntu",
-                "osVersionRegex", "12.*",
+ * "minRam", 8192,
+ * "osFamily", "ubuntu",
+ * "osVersionRegex", "12.*",
  */
```

Comment:
Originally I wanted to force minimum's for these values but @aledsage preferred that we just document the requirement.  I'll extend the documentation to make this clearer.